### PR TITLE
Send release attributes

### PR DESCRIPTION
**Pull request by j0hn**
Sends more metadata on the release request

### CODE REVIEW

File: binstar_client/inspect_package/conda.py [hunk 1]
```diff
@@ -127,25 +127,23 @@ def inspect_conda_package(filename, fileobj, *args, **kwargs):
...
         'summary': about.get('summary', ''),
-        'description': about.get('description', ''),
         'license': about.get('license'),
-        'license_url': about.get('license_url'),
-        'dev_url': about.get('dev_url'),
-        'doc_url': about.get('doc_url'),
-        'home': about.get('home'),
-        'source_git_url': about.get('source_git_url'),
+        'description': about.get('description', ''),
     }
```
Comment (dsludwig) [inline binstar_client/inspect_package/conda.py]: Do we not need to keep these fields on the package for backwards compatibility?

Comment (j0hn) [inline binstar_client/inspect_package/conda.py]: yeah, good point

File: binstar_client/__init__.py [hunk 2]
```diff
@@ -395,17 +399,19 @@ def add_release(self, login, package_name, version, requirements, announce,
         :param version: the version string of the release
         :param requirements: A dict of requirements TODO: describe
         :param announce: An announcement that will be posted to all package watchers
-        :param description: A long description about the package
```
Comment (jakirkham) [inline binstar_client/__init__.py]: Why was `description` dropped?

Comment (j0hn) [inline binstar_client/__init__.py]: `description` is included on `release_attrs`

Comment (jakirkham) [inline binstar_client/__init__.py]: I see. Thanks.

**Pull request merged**